SparseGrids/CMakeLists.txt

```diff
@@ -0,0 +1,266 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Usually only one require at the root of the project.

Reply:
OK, will remove the extra ones.